src/utils/makerdao.ts

```diff
@@ -100,18 +89,30 @@ export async function getMakerDaoData() {
     delay: 1,
   })
 
-  const executiveVotes = extractProposals(topics, network)
+  const allSpells = await promiseRetry({
+    fn: fetchSpells,
+    times: 4,
+    delay: 1,
+  })
+
+  //const executiveVotes = extractProposals(topics, network)
   const historicalPolls = formatHistoricalPolls(topics)
+  const spellsInfo = allSpells.map(({ source, title, proposal_blurb, about }) => ({
+    source,
+    title,
+    proposal_blurb,
+    about,
+  }))
 
-  return { executiveVotes, historicalPolls }
+  return { historicalPolls, spellsInfo }
```

Comment:
I think we could get rid of `historicalPolls` is not used anywhere in the app. 
The rest looks good!!